packages/config/src/cache.ts

```diff
@@ -0,0 +1,48 @@
+import { WalletConfig } from './config'
+
+const maxCachedConfigs = 10
+
+const listKey = '@sequence.config.imageHashes'
+const configKey = (imageHash: string) => `@sequence.config.${imageHash}`
+
+let storage: {
+  getItem(key: string): string | null
+  setItem(key: string, value: string): void
+  removeItem(key: string): void
+}
+
+try {
+  storage = localStorage
```

Comment:
nice, looks good to try localStorage and have fallback.

does this work though..? I wonder if you need `(window as any).localStorage` ..? if this works though, looks great

Reply:
Yeah, so this works, but I'm not sure how idiomatic it is. But config tests without local storage still work, and it runs in browser too. And no `any` cast is necessary here.